magento-research/pwa-studio
# [dev]: Update PR template for danger checks (and update danger)

## Description
Adding some tooling around the contents of PR as in the labels, description, etc:
* Made some small changes to the PR template.
* Updated [dangerfile](https://github.com/magento/pwa-studio-cicd/pull/5) for checking PR labels, description and other rules we want to apply to the pull request metadata rather than the code changes.
* Trigger that job hourly since AWS CodeBuild doesn't allow usage of Github's PR Edit webhooks (currently they only have create/update/reopen).


## Related Issue





Closes #1279 .

## Verification Steps

1. Check out the PR and `yarn install`.
2. Run `danger pr https://github.com/magento-research/pwa-studio/pull/1269` or replace the PR link with some other PR you want to test.
3. See results in your console.
4. To test each failure, open your own PR and:
  * Add a TODO in the description somewhere
  * Delete a section header completely.
  * Leave one of the sections as the default text like `Closes #<ISSUE_NUMBER>`.
  * After the above, replace `#ISSUE_NUMBER` with a closed issue.
  * Do not apply a version label.
  * Check that appropriate project labels are applied (should apply on changes to code within the `packages` directories.

## How have YOU tested this?
Tested locally with `danger pr https://github.com/magento-research/pwa-studio/pull/1269`

## Proposed Labels for Change Type/Package




## Checklist:
